### PR TITLE
Add nginx options to KONFIG object

### PR DESCRIPTION
**Pull request by szkl**
Signed-off-by: Sonmez Kartal <sonmez@koding.com>

### CODE REVIEW

File: deployment/nginx.coffee [hunk 1]
```diff
@@ -460,3 +462,18 @@ createHttpsRedirector = (KONFIG) ->
       # use generic names, do not hardcode values
       return 301 https://$host$request_uri;
     }"""
+
+createHttpOptions = (KONFIG) ->
```
Comment (cihangir) [inline deployment/nginx.coffee]: Could you combine this  and createListenDirective together to have a unified functionality? (we can set the defaults for options.nginx for createListenDirective )

**Review by cihangir — CHANGES_REQUESTED** [review]
unification needed

File: deployment/nginx.coffee (hunk 1 above)
Comment (szkl) [reply deployment/nginx.coffee]: real_ip module directives are set in `http` block. It applies to everywhere functionally. Is this what you are suggesting?

Comment (cihangir) [reply deployment/nginx.coffee]: There is a function in this file called "createListenDirective" which has pretty similar body.

**Comment by szkl** [comment]
Updated.

**Review by cihangir — APPROVED** [review]

**Pull request merged**